api/helpers/game-states/unpack-gamestate.js

```diff
@@ -44,7 +44,7 @@ module.exports = {
             ? value.map((cardStr) => {
               // Is the card frozen
               const freezeCard =
-                  gameStateRow.moveType === MoveType.RESOLVE &&
+                  !!gameStateRow.resolved &&
```

Comment:
This was the key fix. The moveType might be REQUEST_STALEMATE or REQUEST_STALEMATE but still have just resolved a 9